packages/grafana-ui/src/components/Cascader/Cascader.tsx

```diff
@@ -195,7 +199,7 @@ export class Cascader extends React.PureComponent<CascaderProps, CascaderState>
           <RCCascader
             onChange={onChangeCascader(this.onChange)}
             options={this.props.options}
-            changeOnSelect
+            changeOnSelect={changeOnSelect}
```

Comment:
This property name in the rc-cascade component is a bit weird. The documentation doesn't do a good job of describing what it does either.